just-install.json

```diff
@@ -1504,6 +1522,17 @@
       },
       "version": "latest"
     },
+    "rclone": {
+      "installer": {
+        "kind": "zip",
+        "options": {
+          "destination": "{{.PROGRAMFILES}}\\Rclone"
```

Comment:
You might also want to add shims.

Reply:
I contemplated it, I'm not sure how it would work as the directory structure created would be `%programfiles%\Rclone\rclone-version-whatever\rclone.exe`

Do you know if there are any options in the zip library to strip the first directory, ala with `tar`s `--strip 1` option?

Reply:
No, not at the moment. I might implement that if I have time.